airbyte-cdk/python/airbyte_cdk/sources/http_logger.py

```diff
@@ -31,16 +30,13 @@ def format_http_json(response: requests.Response, logger: str):
             },
         },
         "log": {
-            "logger": logger,
             "level": "debug",
         },
         "url": {"full": request.url},
     }
-    return filter_secrets(json.dumps(log_message))
```

Comment:
I see that you moved `filter_secrets` to [where we're emitting the message](https://github.com/airbytehq/airbyte/pull/27990/files#diff-fa683949c758fb9056f1525e9e429eb996c714450ce3b432861bca5a983b1957R78). This makes sense if we always emit these messages from the message repository, but there are still ways to emit logs without going through it. Is there something about this particular type of log message that gives you confidence that they'll always be emitted via the repository?

Reply:
Theoretically we could call `filter_secrets` on all messages including control messages - it would just be a noop if it's not needed. That said, I'm in favor of doing it in the place where it makes the most sense and as long as we're confident that messages that need secrets filtering will be emitted via the message repository I'm good with where you moved it.